apps/admin-gui/src/app/shared/components/roles-list/roles-page.component.html

```diff
@@ -99,16 +110,32 @@ <h1 class="page-subtitle d-flex">{{'ROLES.TITLE'| translate}}</h1>
         <div *ngIf="showDescription">
           {{('ROLES.' + role + '_GROUPS_' + entityType) | translate}}
         </div>
-
+        <span
+          matTooltip="{{'ROLES.REMOVE_DISABLED_TOOLTIP'| translate}}"
+          matTooltipPosition="left"
+          [matTooltipDisabled]="selection.selected.length === 0 || !disableRemove">
+          <button
+            (click)="openConfirmDialog(role)"
+            [disabled]="selection.selected.length === 0 || disableRemove"
+            mat-flat-button
```

Comment:
NITPICK: Maybe something like this looks better?
![image](https://user-images.githubusercontent.com/70371638/195890859-1be94102-abad-4da4-b522-8568bbf7948b.png)
```suggestion
            class="mt-2"
            mat-flat-button
```

instead of this:
![image](https://user-images.githubusercontent.com/70371638/195891010-99316a44-3701-42fe-b500-073bb387d073.png)